pysiaf/aperture.py

```diff
@@ -176,31 +177,35 @@ class Aperture(object):
     Transformations between four coordinate systems ("frames") are supported:
         * `Detector ("det")` :  units of pixels, according to detector read out axes orientation as
         defined by SIAF. This system generally differs from the JWST-DMS detector frame definition.
+
         * `Science ("sci")` :   units of pixels, corresponds to DMS coordinate system.
+
         * `Ideal ("idl")` : units of arcseconds, usually a tangent plane projection with reference
         point at aperture reference location.
+
         * `Telescope or V2/V3 ("tel")` : units of arcsecs, spherical coordinate system.
 
     Examples
     --------
-    ap = some_siaf['desired_aperture_name']     # extract one aperture from a Siaf object
+    extract one aperture from a Siaf object:
+    ``ap = some_siaf['desired_aperture_name']``
 
-    ap.det_to_tel(1024, 512)                    # convert pixel coordinates to V2V3 / tel coords.
-                                                # takes pixel coords, returns arcsec
+    convert pixel coordinates to V2V3 / tel coords (takes pixel coords, returns arcsec):
+    ``ap.det_to_tel(1024, 512)``
 
-    ap.idl_to_sci( 10, 3)                       # convert idl coords to sci pixels
-                                                # takes arcsec, returns pixels
-
-    There exist methods for all of the possible
-    {tel,idl,sci,det}_to_{tel,idl,sci,det} combinations.
+    convert idl coords to sci pixels: ``ap.idl_to_sci( 10, 3)``
 
     Frames can also be defined by strings:
-    ap.convert(1024, 512, from_frame='det', to_frame='tel')  # same as first example above
+    ``ap.convert(1024, 512, from_frame='det', to_frame='tel')``
+
+    Get aperture vertices/corners in tel frame: ``ap.corners('tel')``
+
+    Get the reference point defined in the SIAF: ``ap.reference_point('tel')``
 
-    ap.corners('tel')                           # Get aperture vertices/corners in tel frame
-    ap.reference_point('tel')                   # Get the reference point defined in the SIAF
+    plot coordinates in idl frame: ``ap.plot('tel')``
 
-    ap.plot('tel')                              # plot coordinates in idl frame
+    There exist methods for all of the possible ``{tel,idl,sci,det}_to_{tel,idl,sci,det}``
```

Comment:
I agree in principle. 
I did not include the `raw` frame here because it is not (yet) a documented frame. I feel like it may create more confusion than benefit for the non-expert.

Reply:
Ah okay that makes sense- then yes I agree with waiting to include the raw frame in this list